test/unit/lifecycle/properties.js

```diff
@@ -325,4 +338,21 @@ describe('lifecycle/property', function () {
       });
     });
   });
+  
+  describe('patterns', function () {
```

Comment:
Would be good to test that you have access to `data.internalValue` inside `get()` as well